src/utils/rowTemplateColumns.ts

```diff
@@ -0,0 +1,354 @@
+            { label: 'Windows', key: 'windows' },
+            { label: 'macOS', key: 'osx' },
+            { label: 'Linux', key: 'linux' },
+            { label: 'Android', key: 'android' },
+            { label: 'iOS', key: 'ios' },
+          ],
+          matchesFunction(result: MannWhitneyResultsItem, valueKey: string) {
+            const label = this.possibleValues.find(
+              ({ key }) => key === valueKey,
+            )?.label;
+            const platformName = getPlatformShortName(result.platform);
+            return platformName === label;
+          },
+        },
+      };
+
+  const columnsMannWhitneyConfiguration: CompareMannWhitneyResultsTableConfig =
+    [
+      {
+        ...tableMannWhitneyConfigTestSubtestDiff.platformConfig,
+      },
+      {
+        name: 'Base',
+        key: 'base',
+        gridWidth: '.5fr',
+        tooltip: 'A summary of all values from Base runs using a mean.',
+      },
+      {
+        key: 'comparisonSign',
+        gridWidth: '0.2fr',
+      },
+      {
+        name: 'New',
+        key: 'new',
+        gridWidth: tableMannWhitneyConfigTestSubtestDiff.newGridWidth,
+        tooltip: 'A summary of all values from New runs using a mean.',
+      },
+      {
+        name: 'Status',
+        filter: true,
+        key: 'status',
+        gridWidth: '1fr',
+        possibleValues: [
+          { label: 'No changes', key: 'none' },
+          { label: 'Improvement', key: 'improvement' },
+          { label: 'Regression', key: 'regression' },
+        ],
+        matchesFunction(result, valueKey) {
+          switch (valueKey) {
+            case 'improvement':
+              return result.direction_of_change === 'improvement';
+            case 'regression':
+              return result.direction_of_change === 'regression';
+            default:
+              return (
+                !result.direction_of_change ||
+                result.direction_of_change === 'no change'
+              );
+          }
+        },
+        tooltip:
+          'An improvement or regression being shown here means that the effect size is meaningful, and the difference has a significant p-value.',
+      },
+      {
+        name: "Cliff's Delta",
+        key: 'delta',
+        gridWidth: tableMannWhitneyConfigTestSubtestDiff.cliffsDeltaGridWidth,
+        sortFunction(resultA, resultB) {
+          return (
+            Math.abs(resultA.cliffs_delta) - Math.abs(resultB.cliffs_delta)
+          );
+        },
+        tooltip:
+          'Cliff’s Delta quantifies the magnitude of the difference between Base and New values.',
+      },
+      {
+        name: 'Significance',
+        key: 'significance',
+        filter: true,
+        gridWidth: tableMannWhitneyConfigTestSubtestDiff.significanceGridWidth,
+        tooltip:
+          'Mann Whitney U test p-value indicating statistical significance. Mann Whitney U p-value < .05 indicates a statistically significant difference between Base and New.',
```

Comment:
Change this to the following: `Significance of the comparison as determined by a Mann Whitney U test. A significant comparison has a p-value of less than 0.05.`